core/arch/arm/mm/core_mmu.c

```diff
@@ -158,8 +158,9 @@ static void core_mmu_mmap_init(struct tee_mmap_region *mm, size_t max_elem,
 
 	for (n = 0; map[n].type != MEM_AREA_NOTYPE; n++) {
 		mme.pa = map[n].pa;
-		mme.va = map[n].pa;
+		mme.va = map[n].va;
```

Comment:
I'm refering to line 163 (https://github.com/OP-TEE/optee_os/pull/426/files#diff-21c5a372f38c092a429fd2b7e8f71ef3R163) which is

    mme.map = &map[n];

as `mme.va == mme.map->va`, I am wondering if we need `mme.va` (same for `pa` and `size`)